src/NzbDrone.Core/ImportLists/PTP/PTPImport.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using NLog;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Parser;
+
+namespace NzbDrone.Core.ImportLists.PTP;
+
+public class PTPImport : HttpImportListBase<PTPSettings>
+{
+    public override string Name => "PTP Collection";
```

Comment:
Is this a fine name, or should I expand it in full to "PassThePopcorn Collection"?

Reply:
I like `PassThePopcorn Collection`